PrivateHeaders/APPX/ZIP.h

```diff
@@ -400,25 +407,11 @@ namespace appx {
 
         std::cout << "\n\n" << manifestText << "\n\n";
 
-        std::size_t manifestTextSize = manifestText.size();
-        const std::uint8_t *manifestTextBytes =
-            reinterpret_cast<const std::uint8_t *>(manifestText.c_str());
-        std::uint32_t crc32;
-        SHA256Hash sha256;
-        {
-            CRC32Sink crc32Sink;
-            SHA256Sink sha256Sink;
-            auto sink = MakeMultiSink(crc32Sink, sha256Sink);
-            sink.Write(manifestTextSize, manifestTextBytes);
-            crc32 = crc32Sink.CRC32();
-            sha256 = sha256Sink.SHA256();
-        }
-        assert(manifestText.size() < std::numeric_limits<off_t>::max());
-        ZIPFileEntry entry(archiveFileName, static_cast<off_t>(manifestTextSize),
-                           offset, crc32, {}, sha256);
-        entry.WriteFileRecordHeader(sink);
-        sink.Write(manifestTextSize, manifestTextBytes);
-        return entry;
+        std::string outputFileName = inputFileName + ".temp";
+        std::ofstream manifestOutput(outputFileName);
```

Comment:
This is bad. We should not write temp files like this. Let me know if you need help writing a compressed string as a ZIP file entry.